tools/db_bench_tool.cc

```diff
@@ -2576,14 +2581,15 @@ class Benchmark {
     fprintf(stderr, "RocksDB:    version %d.%d\n",
             kMajorVersion, kMinorVersion);
 
-#if defined(__linux)
+#if defined(__linux__) || defined(__APPLE__)
     time_t now = time(nullptr);
     char buf[52];
     // Lint complains about ctime() usage, so replace it with ctime_r(). The
     // requirement is to provide a buffer which is at least 26 bytes.
     fprintf(stderr, "Date:       %s",
             ctime_r(&now, buf));  // ctime_r() adds newline
 
+#if defined(__linux)
     FILE* cpuinfo = fopen("/proc/cpuinfo", "r");
```

Comment:
The code varies between using __linux and __linux__.  Probably should be one or the other...